scripts/target_determinator.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+"""Computes the potentially differing rules from some git commit.
+
+Wraps the "target-determinator" Go program here:
+https://github.com/bazel-contrib/target-determinator
+
+The purpose is to compute the potentially impacted set of targets from some
+provided Git commit to the current checkout.
+
+This script will ensure a cached version of the latest release is available, and
+then forward a limited set of flags to it. This script also filters the
+resulting targets using `bazel query` to make it the most relevant list for
+continuous integration.
+"""
+
+__copyright__ = """
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""
+
+import subprocess
+import argparse
+import tempfile
+import sys
+from pathlib import Path
+from typing import List
+
+import scripts_utils
+
+
+def log(s: str) -> None:
+    print(s, file=sys.stderr)
+
+
+def quiet_run_output(args: List[str]) -> str:
```

Comment:
This is only called once now, maybe it can be removed? Partly, I think the name makes less sense when quiet_run is gone.

Reply:
Sure, done.